pkg/volume/csi/csi_plugin_test.go

```diff
@@ -1505,3 +1513,55 @@ func TestHighestSupportedVersion(t *testing.T) {
 		}
 	}
 }
+
+func TestToPluginHandler(t *testing.T) {
+	testCases := map[string]struct {
+		genericPlugin  volume.VolumePlugin
+		expectedErrMsg string
+	}{
+		"nil cannot be type-casted to csiPlugin": {
+			genericPlugin:  nil,
+			expectedErrMsg: "is not a CSI volume plugin",
+		},
+		"a generic plugin cannot be type-casted to csiPlugin": {
+			genericPlugin:  &volumetest.FakeVolumePlugin{},
+			expectedErrMsg: "is not a CSI volume plugin",
+		},
+		"a CSI plugin can be used as a PluginHandler": {
+			genericPlugin:  &csiPlugin{},
+			expectedErrMsg: "",
+		},
+	}
+
+	for tcName, tc := range testCases {
+		t.Run(tcName, func(t *testing.T) {
+			plug, err := ToPluginHandler(tc.genericPlugin)
```

Comment:
Where is function `ToPluginHandler` defined?  Is it not in this PR?  I don't see it.

Reply:
https://github.com/kubernetes/kubernetes/pull/73919/files/351b83aecaa6ba54a5b3afaed3dd3e52aae366f3#diff-f52821ec6236e86b02ca334533395c91R997

It's part of that PR, [you commented on it](https://github.com/kubernetes/kubernetes/pull/73919#discussion_r268868056) ;)